model/task/task.go

```diff
@@ -1469,7 +1469,15 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 		"execution": t.Execution,
 		"project":   t.Project,
 		"details":   t.Details,
+		"status":    detail.Status,
 	})
+	if detail.Status == "" {
+		grip.Error(message.Fields{
```

Comment:
Can we use grip.Error(message.WrapError ? This handles the stack but formats it better.
I would also suggest adding a "message" of some sort (ticket number or "empty task details") to make it easier to create an alert on this, and adding a "task_id" section.